app/assets/scripts/components/documents/document-download-menu.js

```diff
@@ -44,7 +44,9 @@ export default function DocumentDownloadMenu(props) {
     const { id, version, alias } = atbd;
     const pdfUrl = `${apiUrl}/atbds/${id}/versions/${version}/pdf`;
     const pdfFileName = `${alias}-v${version}.pdf`;
-    const maxRetries = 50;
+    // maxRetries * waitBetweenTries = time before user is given an error toast
+    // 30 * 5000 = 150s
+    const maxRetries = 30;
```

Comment:
@sunu had a good point that if the PDF is failing for other reasons, it would take 50*5s to be notified. Now, they will be notified after ~150s.